src/framework/components/collision/component.js

```diff
@@ -132,6 +134,12 @@ Object.assign(pc, function () {
             }
         },
 
+        onSetOffset: function (name, oldValue, newValue) {
+            if (oldValue !== newValue) {
```

Comment:
This won't work for two `pc.Vec3` instances. Instead, you'd need to use `pc.Vec3#equals`.